metaInformation.json

```
@@ -43,5 +43,10 @@
         "name": "OOBE - DFO patches",
         "order": -8,
         "locked": false
-    }
+    },
+	{	
+		"name": "KeyShot",
+		"order": -7,
+		"locked": true
+	}
```

```suggestion
    },
    {	
	"name": "KeyShot",
	"order": -5,
	"locked": true
    }
```
I suggest we change the order to `-5` just to give us some more breathing space for OOBE in the future 😉